### ISSUE TITLE
Learning Adaptive Hierarchical Cuboid Abstractions of 3D Shape Collections

### ISSUE THREAD

**Issue by j20232** [j20232/survey#63]
![image](https://user-images.githubusercontent.com/13940471/89708051-de940180-d9ae-11ea-8d8b-43c393adc369.png)

![image](https://user-images.githubusercontent.com/13940471/89708076-0d11dc80-d9af-11ea-830c-95d832098b37.png)

## Summary
- Adaptive hierarchical cuboid representation for 3D shape abstraction and introduce an unsupervised learning approach to extract this representation from unlabeled shape data.

## Abstract
Abstracting man-made 3D objects as assemblies of primitives, i.e., shape abstraction, is an important task in 3D shape understanding and analysis. In this paper, we propose an unsupervised learning method for automatically constructing compact and expressive shape abstractions of 3D objects in a class. The key idea of our approach is an adaptive hierarchical cuboid representation that abstracts a 3D shape with a set of parametric cuboids adaptively selected from a hierarchical and multi-level cuboid representation shared by all objects in the class. The adaptive hierarchical cuboid abstraction offers a compact representation for modeling the variant shape structures and their coherence at different abstraction levels. Based on this representation, we design a convolutional neural network (CNN) for predicting the parameters of each cuboid in the hierarchical cuboid representation and the adaptive selection mask of cuboids for each input 3D shape. For training the CNN from an unlabeled 3D shape collection, we propose a set of novel loss functions to maximize the approximation quality and compactness of the adaptive hierarchical cuboid abstraction and present a progressive training scheme to refine the cuboid parameters and the cuboid selection mask effectively.
We evaluate the effectiveness of our approach on various 3D shape collections and demonstrate its advantages over the existing cuboid abstraction approach. We also illustrate applications of the resulting adaptive cuboid representations in various shape analysis and manipulation tasks.

## Author
- Chun-Yu Sun 12
- Qian-Fang Zou 23
- Xin Tong 2
- Yang Liu 2

- 1 Tsinghua University
- 2 Microsoft Research Asia
- 3 University of Science and Technology of China

## Journal/Conference
SIGGRAPH Asia 2019

## Link
- ACM: https://dl.acm.org/doi/10.1145/3355089.3356529
- Official: https://isunchy.github.io/projects/cuboid_abstraction.html
- Code: https://github.com/isunchy/cuboid_abstraction
